apiserver/apiserver/apiserver.py

```diff
@@ -33,6 +33,7 @@ def status_response(status):
 
 @routes.get('/healthcheck')
 async def healthcheck(request):
+    del request
```

Comment:
What is the role of this?

Reply:
pacify pylint http://pylint-messages.wikidot.com/messages:w0613